### ISSUE TITLE
Accessibility: Typo in aria attribute valuemin for InputNumber

### ISSUE THREAD

**Issue by riorudo** [primefaces/primeng#10058]
There is a typo for the aria attribute `valuemin` in the component [p-inputNumber](https://github.com/primefaces/primeng/blob/master/src/app/components/inputnumber/inputnumber.ts).

`[attr.aria-valumin]="min"` instead of `[attr.aria-valuemin]="min"` ("e" is missing)

**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**

[attr.aria-valumin]="min" 

**Expected behavior**

[attr.aria-valu**e**min]="min"

* **PrimeNG version:** 11.3.1
